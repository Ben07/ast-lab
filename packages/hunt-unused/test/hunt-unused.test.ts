import huntUnused from '../src';

describe('huntUnused', () => {
  test('fixture test', async () => {
    expect(
      await huntUnused(['test/fixtures/index.js'], {
        source: 'test/fixtures/*.js',
      })
    ).toMatchSnapshot();
  });
});
