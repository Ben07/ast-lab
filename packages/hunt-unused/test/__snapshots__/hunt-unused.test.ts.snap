// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`huntUnused fixture test 1`] = `
Object {
  "/Users/jennie.ji/Projects/affected/packages/hunt-unused/test/fixtures/b.js": Array [
    "b2",
  ],
  "/Users/jennie.ji/Projects/affected/packages/hunt-unused/test/fixtures/c.js": Array [],
  "/Users/jennie.ji/Projects/affected/packages/hunt-unused/test/fixtures/d.js": Array [
    "*",
  ],
}
`;
